src/alpaca.erl

```diff
@@ -36,13 +41,19 @@
 -include_lib("eunit/include/eunit.hrl").
 -endif.
 
+-define(COMPILER_VERSION, "0.2.7").
```

Comment:
Just thought, we might want to already call this `"0.2.8"` as that's the next release and `0.2.7` is already tagged.